### ISSUE TITLE
Secondry x_labels

### ISSUE THREAD

**Issue by marcinkubica** [Kozea/pygal#272]
Hi All
Good work on this lib gentlemen. Gotta say I really like it.

Any chance you could shed some light how to get secondary x label working? Basically I just need to have extra data displayed under each X label.

Any tips or hacks?

Many thanks
Marcin

**Comment by mehmettaskiner** [comment]
+1

**Comment by paradoxxxzero** [comment]
What extra data do you want to display? Do you have an example?
The closest I could think of is [x-labels](http://pygal.readthedocs.org/en/latest/documentation/configuration/label.html#x-labels).
As for now if you want to add more to the display you might have to inherit the chart and add some rendering.

**Comment by marcinkubica** [comment]
Hi @paradoxxxzero 

My use case at that time was I needed to display percentages of KPI in a stacked bar, and on X labels I needed to have days of Week, and under that a values for each day which was a total of 'units' on which each bar had KPI percentages.

**Comment by paradoxxxzero** [comment]
So you want a way to break lines in labels?

**Comment by marcinkubica** [comment]
@paradoxxxzero yes that was the idea, break the lines and allow to pass another values in.
